phoenix-core/src/it/java/org/apache/phoenix/end2end/index/GlobalIndexCheckerIT.java

```diff
@@ -169,6 +171,48 @@ public void testDeleteNonExistingRow() throws Exception {
         }
     }
 
+    @Test
+    public void testIndexRowWithoutEmptyColumn() throws Exception {
+        if (async) {
+            // No need to run the same test twice one for async = true and the other for async = false
+            return;
+        }
+        long scn;
+        String dataTableName = generateUniqueName();
+        String indexTableName = generateUniqueName();
+        try (Connection conn = DriverManager.getConnection(getUrl())) {
+            populateTable(dataTableName); // with two rows ('a', 'ab', 'abc', 'abcd') and ('b', 'bc', 'bcd', 'bcde')
+            Thread.sleep(1000);
```

Comment:
This can be done with EnvironmentEdgeManager rather than a sleep, and that's faster/safer than a sleep. Just remember to increment by at least one after any DDL or DML to be able to read it.

Reply:
Sleep is not necessary. I forgot to remove it. Thank you for catching it